docs/extras/modules/data_connection/vectorstores/integrations/vectara.mdx

```diff
@@ -6,7 +6,7 @@ import CodeBlock from "@theme/CodeBlock";
 
 # Vectara
 
-Vectara is a developer-first API platform for easily building conversational search experiences.
+Vectara is a platform for building GenAI applications. It provides an easy-to-use API for document indexing and query that is managed by Vectara and is optimized for performance and accuracy.
```

Comment:
Typo: "document indexing and query that is managed..."

Reply:
Thanks for noticing this! Should be fixed now